modAionImpl/src/org/aion/zero/impl/db/AionBlockStore.java

```diff
@@ -63,6 +64,9 @@
 
     private boolean checkIntegrity = true;
 
+    private Stack<IAionBlock> branchingBlk, preBranchingBlk;
```

Comment:
Using the [Deque](https://docs.oracle.com/javase/10/docs/api/java/util/Deque.html) class is preferable to the use of Stack as described in the JavaDoc.